gcloud/bigquery/client.py

```diff
@@ -96,3 +97,20 @@ def dataset(self, name):
         :returns: a new ``Dataset`` instance
         """
         return Dataset(name, client=self)
+
+    def load_from_storage(self, name, destination, *source_uris):
```

Comment:
I don't think it needs to:  the method name describes what the user wants, and returns a `job` instance which captures that intent.